Tasks/PublishTestResultsV2/cieventlogger.ts

```diff
@@ -13,8 +14,17 @@ function getDefaultProps() {
     };
 }
 
-export function publishEvent(properties: { [key: string]: any }): void {
+export function addToConsolidatedCi(key: string, value: any) {
+    consolidatedCiData[key] = value;
+}
+
+export function fireConsolidatedCi() {
```

Comment:
add a call to this in a finally block around the main code

Reply:
Done